docs/_includes/input-chips.html

```diff
@@ -0,0 +1,344 @@
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-1-3-text" class="chip__text">Baseball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-3-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                        </div>
+                        <button type="button" class="icon-btn icon-btn--small input-chips_list-toggle">
+                            <svg class="icon icon--chevron-down-16" focusable="false" height="16" width="16" aria-hidden="true">
+                                {% include symbol.html name="chevron-down-16" %}
+                            </svg>
+                        </button>    
+                    </div>
+                    <span class="combobox combobox--js input-chips__combobox">
+                        <span class="combobox__control input-chips_combobox__control">
+                            <input id="input-chips-1-input" role="combobox" type="text" placeholder="Add Sport" aria-haspopup="listbox" aria-owns="listbox-input-chips-1" />
+                        </span>
+                        <div class="combobox__listbox">
+                            <div id="listbox-input-chips-1" class="combobox__options" role="listbox">
+                                <div class="combobox__option" role="option">
+                                    <span>Baseball</span>
+                                </div>
+                                <div class="combobox__option" role="option">
+                                    <span>Hockey</span>
+                                </div>
+                                <div class="combobox__option" role="option">
+                                    <span>Tennis</span>
+                                </div>
+                            </div>
+                        </div>
+                    </span>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {% highlight html %}
+TBD
+    {% endhighlight %}
+
+    <h3>Input Chips with Helper Text</h3>
+    <p>Helper text can be included for additional context or helpful tips.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="input-chips">
+                <label id="input-chips-1-heading" for="input-chips-1-input">Sports</label>
+                <div class="input-chips__container">
+                    <div class="input-chips__content">
+                        <div class="input-chips_items" role="group" aria-describedby="input-chips-1-heading">
```

Comment:
Do we need this extra layer? It seems like we can wrap that in the parent as well.

Reply:
I'll take a look.

Reply:
Actually, `input-chips_items` is necessary since there's a flex there - the group of chips have to be grouped and the toggle button has to be separate.

Reply:
This is what happens we we flatten. 
<img width="1310" alt="image" src="https://github.com/eBay/skin/assets/1675667/30813c59-7315-4269-bea7-5d4bcc52bdb8">

Reply:
I mean is this without changing any styles? 
The way I see it is if we do:
input-chips root container (should display flex, column I think)
    input-chips-container (contains all chips, display flex, row)
    combobox input
I would think that should work fine.